migration/v0.20.x-v0.21.x/data-stream-rollovers.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
Please add this script to the markdown file as well so that we remember to run it